conf/modules.config

```diff
@@ -278,6 +278,35 @@ process {
         ]
     }
 
+    // Reference elongation and indexing for circular mapping
+    withName: GUNZIP_ELONGATED_FASTA {
+        publishDir = [
+            path: { "${params.outdir}/reference/${meta.id}_${params.mapping_circularmapper_elongation_factor}/" },
+            mode: params.publish_dir_mode,
+            pattern: '*[0-9].f*',
+            enabled: params.save_reference
+        ]
+    }
+
+    withName: CIRCULARMAPPER_CIRCULARGENERATOR {
+        tag = { "${meta.id}_${params.mapping_circularmapper_elongation_factor}" }
+        publishDir = [
+            path: { "${params.outdir}/reference/${meta.id}_${params.mapping_circularmapper_elongation_factor}/" },
+            mode: params.publish_dir_mode,
+            pattern: '*[0-9].fasta',
```

Comment:
Would that work exactly? what is the fasta has the GenBank accession in it? I think NXF should anyway exclude input file from output glob patterns